client/src/main/java/se/fortnox/reactivewizard/client/HttpClient.java

```diff
@@ -277,6 +277,19 @@ private <T> Mono<T> convertError(RequestBuilder fullReq, Throwable throwable) {
         return Mono.error(throwable);
     }
 
+    private static void logFailedRequest(Throwable throwable, String request) {
+        var logAsInfo = throwable instanceof WebException webException &&
```

Comment:
I remember now that you prefer more verbose style :) 

What do you mean by "short hand"? 

Also how do you suggest that I avoid using not? Your suggestion won't work out of the box since WARN should be the default. I guess I could and a case assigning WARN again if it is "resource.not.found" but not sure that is cleaner?